### PR TITLE
config: bump jgit to 5.5.1.201910021850

### PR DESCRIPTION
**Pull request by romani**
https://travis-ci.org/checkstyle/checkstyle/jobs/598822377#L1069

### CODE REVIEW

**Comment by romani** [comment]
Needed from phone

File: pom.xml [hunk 1]
```diff
@@ -197,7 +197,7 @@
     <maven.spotbugs.plugin.version>3.1.12.2</maven.spotbugs.plugin.version>
     <maven.pmd.plugin.version>3.12.0</maven.pmd.plugin.version>
     <pmd.version>6.17.0</pmd.version>
-    <maven.jacoco.plugin.version>0.8.4</maven.jacoco.plugin.version>
+    <maven.jacoco.plugin.version>0.8.5</maven.jacoco.plugin.version>
```
Comment (rnveach) [inline pom.xml]: Jacoco should be in its own commit, not in the same commit as jgit.

Comment (romani) [reply pom.xml]: Yes, by bad, it was a commit from phone. I will do it properly, but good that all CI pass, so I can merge it  after commit clean up

Comment (rnveach) [reply pom.xml]: I have split the commits into 2.

**Comment by romani** [comment]
This PR is broken, it should not be done from forked master, and first commit is already in upstream master. 

I just need laptop time to fix my fork, and do proper commit/PR
